pydoctor/test/test_astbuilder.py

```diff
@@ -1681,3 +1681,113 @@ def func():
     ''', systemcls=systemcls)
     outer = mod.contents['outer']
     assert not outer.contents
+
+@systemcls_param
+def test_constant_module(systemcls: Type[model.System]) -> None:
+    """
+    It can recognize constants define as module members.
+    """
+    mod = fromText('''
+    LANG = 'FR'
+    ''', systemcls=systemcls)
+    assert getattr(mod.contents['LANG'], 'kind') == model.DocumentableKind.CONSTANT
```

Comment:
```suggestion
    lang = mod.contents['LANG']
    assert isinstance(lang, model.Attribute)
    assert lang.kind is model.DocumentableKind.CONSTANT
```
This provides mypy with the information it needs to actually check the types, unlike `getattr()` which returns `Any`.

Also you can use `is` instead of `==` to compare enum members.

Reply:
Cmon!

Reply:
The main point of type-checking the tests is to have additional checks on whether the annotations in the code under test are correct. By using `getattr()`, we lose that advantage. That doesn't mean you can't ever use `getattr()`, but in  this case it can be avoided without too much hassle.